src/main/kotlin/org/jetbrains/research/testgenie/evosuite/Pipeline.kt

```diff
@@ -280,6 +280,7 @@ class Pipeline(
         val evoSuiteProcess = GeneralCommandLine(cmd)
         evoSuiteProcess.charset = Charset.forName("UTF-8")
         evoSuiteProcess.setWorkDirectory(projectPath)
+        var isUnknownClass = false
```

Comment:
Yes, I agree that creating such variables is a bad idea, I just couldn't catch other error messages in `EvoSuite` that don't show up in `handler.ExitCode`, and considered this as an unique case.
I implemented a more extensible error handling, for this I created a data class and two variables – a condition for the presence of an error and a message about it. And because didn't find any other examples, I made all `EvoSuite` errors in one style.
I still have problems with the fact that I need to pass the `exitCode` to the message if it is not equal to `0` and `evoSuiteProcessTimeout`. There are 3 options: create a separate class to interact with such errors and `.properties`, add them to the message already in the code, or not add them at all. What is the best way to do this? I added `TODO`s about it.